data/hai/hai outfits.txt

```diff
@@ -549,6 +549,50 @@ outfit `"Bufaer" Atomic Thruster`
 	"flare sound" "atomic large"
 	description "These thrusters are the largest of the Hai atomic thrusters, and one of the most powerful thrusters in known space. Beat out by only the largest human atomic thrusters, these massive engines provide enough thrust to push around even the largest of ships as if they were fighters."
 
+outfit `"Basrem" Reverse Thruster`
+	category "Engines"
+	"cost" 150000
+	thumbnail "outfit/tiny atomic reverse hai"
+	"mass" 18
+	"outfit space" -18
+	"weapon capacity" -18
+	"reverse thrust" 12.5
+	"reverse thrusting energy" 1.4
+	"reverse thrusting heat" 2.8
+	"reverse flare sprite" "effect/atomic flare/tiny"
+		"frame rate" 14
+	"reverse flare sound" "atomic tiny"
+	description "This tiny reverse thruster allows even the smallest of Hai ships to mount them for additional maneuverability beyond normal steering. Many young Hai find these an invaluable investment when first setting out on their own."
```

Comment:
```suggestion
	description "This tiny reverse thruster allows even the smallest of Hai ships additional maneuverability beyond normal steering. Many young Hai find these an invaluable investment when first setting out on their own."
```

First two depending on which way you want to go on number agreement there (but plural would be most consistent with other entries), but honestly it's a bit awkward either way; third just skips the problem entirely and is far smoother.

You could also trim the phrase and still switch it to plural to be consistent with other entries, if you prefer that.